src/site/apt/index.apt.vm

```diff
@@ -166,6 +166,37 @@ Welcome to ${project.name}!
 </project>
 +-----+
 
+** GoogleAnalytics
+
+ Users can optionally enable <<Google Analytics>> for the generated site:
+
++-----+
+<project name="xxx">
+  [...]
+  <googleAnalytics>UA-12345...</googleAnalytics>
+  [...]
+</project>
++-----+
+
+It is also possible to enable the options to force SSL or anonymize the IP address:
```

Comment:
This does not make any sense now because it is on by default. Can you reword, e,g, both are on by default and it is recommended not to change...?